package.json

```diff
@@ -46,7 +46,6 @@
     "vt-pbf": "^3.1.3"
   },
   "devDependencies": {
-    "autoprefixer": "^10.4.19",
```

Comment:
Any idea why this `autoprefixer` dep shifted around? Minor, but would be nice to avoid this to keep the diff as small as possible.

Reply:
I think `npm` did this on its own, but you're right.